packages/button/src/spectrum-clear-button.css

```diff
@@ -52,7 +52,12 @@ THIS FILE IS MACHINE GENERATED. DO NOT EDIT */
         var(--spectrum-alias-icon-color)
     );
 }
-.spectrum-ClearButton--small {
+#button > .spectrum-Icon {
+    /* .spectrum-ClearButton>.spectrum-Icon */
+    margin: 0 auto; /* .spectrum-ClearButton>.spectrum-Icon */
+    margin: 0;
```

Comment:
Will `margin: 0` override `margin: 0 auto`?

Reply:
Hrm. Good catch. Looks like this is mangling both a standard selector and one in a media query. But also, not really applied because of `.spectrum-Icon` vs `sp-icon`... Will review.

Reply:
Not sure how important it is to update this being the two values for margin end up with the same output, generally:
![image](https://user-images.githubusercontent.com/1156657/81229565-080f4c80-8fbe-11ea-81ca-cf71ded2e1e2.png)

![image](https://user-images.githubusercontent.com/1156657/81229588-12314b00-8fbe-11ea-8249-dc24b0c609d3.png)

And the work to update the PostCSS plugin to better support `@media` selectors is being tracked already via: https://github.com/adobe/spectrum-web-components/issues/593

Please, let me know if you can think of anything that we should take into account while de-prioritizing this.

Reply:
I did, however, correct the icon selector here.

Reply:
Ah I see, if PostCSS plugin work is being tracked separately, I'm completely fine with de-prioritizing it here 👍